docker-gc

```diff
@@ -283,3 +306,35 @@ else
     xargs -n 1 $DOCKER rmi $FORCE_IMAGE_FLAG < images.reap &>/dev/null || true
 fi
 
+if [[ $REMOVE_VOLUMES -gt 0 ]]; then
+    set +e
+    $DOCKER volume ls --filter "dangling=true" -q &> /dev/null
+    VOLUMES=$?
+    set -e
+    # If docker volume ls fails, then is probably not supported by either client or server
+    if [[ $VOLUMES -gt 0 ]]; then
+        set +e
+        VERSION=$($DOCKER version --format="Client: {{.Client.Version}} Server: {{.Server.Version}}" 2&>/dev/null)
+        FORMAT=$?
+        set -e
+        if [[ $FORMAT -gt 0 ]]; then
+            log "Removing volumes is not supported for Docker < 1.9.0"
+        else
+            MESSAGE="Removing volumes is not supported for for docker version "$($DOCKER version --format="Client: {{.Client.Version}} Server: {{.Server.Version}}") &
+            log "$MESSAGE"
+        fi
+    else
+        if [[ -z "$VOLUME_DELETE_ONLY_DRIVER" ]]; then
+            $DOCKER volume ls --filter "dangling=true" -q | grep -v -f $EXCLUDE_VOLUMES_IDS_FILE > volumes.reap
```

Comment:
Great, thanks @rccrdpccl .  LGTM -- that was my one suggested change.  Now let's get this in!  ping @sgerrand